<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-3BKRARD" failures="1" tests="3" name="REGISTER_USER_TEST" time="70.543" errors="0" timestamp="2023-04-05T01:00:58 IST">
  <testcase classname="Test.newTest" name="login3" time="20.934">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at Test.newTest.login3(newTest.java:54)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 19 stack frames]]>
    </failure>
  </testcase> <!-- login3 -->
  <testcase classname="Test.newTest" name="loginTest1" time="16.105"/>
  <testcase classname="Test.newTest" name="loginTest2" time="15.542"/>
</testsuite> <!-- REGISTER_USER_TEST -->
